lib/controller/StudyRequestController.js

```diff
@@ -176,6 +176,12 @@ StudyRequestController.push({
     if (studyRequestNew.userSubject !== studyRequestOld.userSubject) {
       return Boom.badRequest('cannot change owner for study request');
     }
+    if (studyRequestNew.assignedTo !== studyRequestOld.assignedTo && !isSupervisor) {
```

Comment:
use `Boom.forbidden()` for these use cases - at this point, these are semantically valid requests, it's just that the user doesn't have permission to do them.